src/Common/src/TypeSystem/IL/ILImporter.cs

```diff
@@ -195,6 +195,12 @@ private void FindJumpTargets()
                         SkipIL(4);
                         break;
                     case ILOpcode.prefix1:
+                        if (_currentOffset >= _ilBytes.Length)
```

Comment:
Should these checks rather be in the ReadILByte, SkipIL, etc. methods so that they kick in everywhere?